docs/en_US/Compressor/Benchmark.md

```diff
@@ -0,0 +1,88 @@
+To provide an initial insight into the performance of various channel pruning algorithms, 
```

Comment:
better to add title for this doc, for example "Comparison of Pruning Algorithms"

Reply:
> better to add title for this doc, for example "Comparison of Pruning Algorithms"
Title 'Comparison of Filter Pruning Algorithms' added